### ISSUE TITLE
Example in eks-cluster-with-new-vpc doesn't work

### ISSUE THREAD

**Issue by ruzickap** [aws-samples/aws-eks-accelerator-for-terraform#79]
It looks like the example in the `deploy/eks-cluster-with-new-vpc` doesn't work:

```bash
git clone https://github.com/aws-samples/aws-eks-accelerator-for-terraform
cd aws-eks-accelerator-for-terraform/deploy/eks-cluster-with-new-vpc
terraform init
terraform apply
...
╷
│ Error: Invalid function argument
│
│   on ../../kubernetes-addons/keda/locals.tf line 75, in locals:
│   75:   default_keda_helm_values = [templatefile("${path.module}/keda-values.yaml", {
│   76:     keda-sa-name = local.keda_service_account_name
│   77:   })]
│     ├────────────────
│     │ path.module is "../../kubernetes-addons/keda"
│
│ Invalid value for "path" parameter: no file exists at ../../kubernetes-addons/keda/keda-values.yaml; this function works only with files that are distributed as part of the
│ configuration source code, so if this file will be created by a resource in this configuration you must instead obtain this result from an attribute of that resource.
╵
╷
│ Error: Reference to undeclared local value
│
│   on ../../kubernetes-addons/prometheus/locals.tf line 71, in locals:
│   71:     ampWorkspaceUrl    = local.amp_workspace_url
│
│ A local value with the name "amp_workspace_url" has not been declared.
```

Thanks for looking at it...

**Comment by vara-bonthu** [comment]
Thanks @ruzickap for raising this issue. Recent merge broke some functionality which i fixed now. I will merge bug fix PR soon

**Comment by askulkarni2** [comment]
Fixed in #80. Closing this issue.
